src/txmempool.cpp

```diff
@@ -890,6 +895,18 @@ void CTxMemPool::ClearPrioritisation(const uint256& hash)
     mapDeltas.erase(hash);
 }
 
+std::map<uint256, std::pair<CAmount, bool>> CTxMemPool::GetPrioritisationMap() const
+{
+    AssertLockNotHeld(cs);
+    LOCK(cs);
+    std::map<uint256, std::pair<CAmount, bool>> result;
```

Comment:
Having `struct delta_info { uint256 txid, CAmount delta, bool in_mempool }` and returning a `vector<delta_info>` might make this a little quicker (using `reserve()` upfront would avoid memory allocations on each iteration).

Reply:
Done